third-party/realdds/include/realdds/dds-adapter-watcher.h

```diff
@@ -0,0 +1,40 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+#pragma once
+
+#include <rsutils/os/adapter-watcher.h>
+#include <set>
+#include <string>
+
+
+namespace realdds {
+
+
+namespace detail {
```

Comment:
detail?

Reply:
Yeah, impl. Usually libraries have a `detail` namespace to indicate this is implementation and subject to change.

Reply:
That's true to every function implementation.
What is the advantages of adding it?

Reply:
NVM